src/interchange/toml/lexer.zig

```diff
@@ -1187,6 +1609,38 @@ pub const Lexer = struct {
     }
 };
 
+pub inline fn isDigit(code_point: CodePoint) bool {
+    return code_point >= '0' and code_point <= '9';
+}
+
+// SAFETY: CodePoints passed here are assumed to be in the range ['0', '9'].
+pub inline fn shiftLeftBase10(comptime P: type, code_point: CodePoint, shift_by: u8) P {
```

Comment:
```suggestion
/// Note: code_point must be a digit.
inline fn shiftLeftBase10(comptime P: type, code_point: CodePoint, shift_by: u8) P {
    bun.debugAssert(isDigit(code_point));
```

1. A lot of these functions might not need to be `pub`?
2. If you use `///` rather than `//`, the comment is treated as a docblock and will show up in editors.
3. You can always add an assert as an extra safety guard in `Debug` mode.

Reply:
Yeah, agree on the pub, I was just following suit with the existing code, although it’s kind of a hot mess. 

Didn’t know about the debug assert! Nice!